src/http-statement-deletion.md

```diff
@@ -0,0 +1,181 @@
+total | Total number of statements found for deletion at initialise
+processing | Boolean value; is there a deletion batch currently being actioned
+done | Boolean value; has the job finished or been terminated
+createdAt | When this document was created
+updatedAt | When this document was last updated
+
+_The `filter` field is stored as text to account for `.` (dot) characters used in query keys - these are invalid in Mongo JSON structures. 
+
+
+#### Example
+
+```json
+{
+  "_id": "111aaa1111a111111aa11111",
+  "organisation": "111aaa1111a111111aa11111",
+  "filter": "{\"statement.verb.id\": \"http://adlnet.gov/expapi/verbs/completed\"}",
+  "pageSize": 1000,
+  "deleteCount": 100000,
+  "total": 100000,
+  "processing": false,
+  "done": true,
+  "createdAt": "2019-01-01T00:00:00Z",
+  "updatedAt": "2019-01-01T00:00:00Z"
+}
+```
+
+The Batch Delete model may be retrieved using the GET [REST](../http-rest) or [Connection APIs](../http-connection) but the other HTTP methods are disabled and instead replaced by specific routes to initialise and terminating the batch deletion jobs.
+
+#### Examples: (using Connection API)
+
+_Note; query parameters should be URL encoded - these examples have had this removed for readability_
+
+##### Fetch a particular job by `_id`
+
+```
+GET http://www.example.org/api/connection/batchdelete?filter={"_id":{"$oid":"111aaa1111a111111aa11112"}}
+Authorization: Basic YOUR_BASIC_AUTH
+Content-Type: application/json; charset=utf-8
+```
+
+##### Fetch the 5 most recently completed/terminated job
+```
+GET http://www.example.org/api/connection/batchdelete?filter={"done":true}&sort={"updatedAt":-1, "_id": 1}&first=5
+Authorization: Basic YOUR_BASIC_AUTH
+Content-Type: application/json; charset=utf-8
+```
+
+##### Fetch the 5 most recently created and unfinished jobs 
+```
+GET http://www.example.org/api/connection/batchdelete?filter={"done":false}&sort={"createdAt":-1, "_id": 1}&first=5
+Authorization: Basic YOUR_BASIC_AUTH
+Content-Type: application/json; charset=utf-8
+```
+
+
+
+### Deletion time window
+Because deletion can be a intensive job for the database, it is possible to configure Learning Locker to only trigger and process batch deletion during a specified window everyday. This can be configured to enable deletions during periods of known low activity (e.g. night time).
+
+To configure this, update the document in the `siteSettings` collection with the required UTC hour, minute and duration:
+
+```mongo
+db.siteSettings.updateOne(
+  {
+    "_id" : ObjectId("111111111111111111111111")
+  },
+  {
+    $set: {
+      batchDeleteWindowStartUTCHour: 00,
+      batchDeleteWindowUTCMinutes: 00,
+      batchDeleteWindowDurationSeconds: 18000
+    }
+  }
+)
+```
+
+The configuration above would allow deletions to be triggered from midnight to 5am (UTC) everyday. Note that the Scheduler process is required to be run in order outstanding jobs at the start of the window everyday.
```

Comment:
```suggestion
The configuration above, for example, would allow deletions to be triggered from midnight to 5am (UTC) everyday. Note that the Scheduler process is required to be run in order for outstanding jobs to be processed at the start of the window every day.
```